### ISSUE TITLE
[A11y] Desktop/Mobile Search Results - Grid: Image Thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1815]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results - Grid: https://staging.oregondigital.org/catalog?locale=en&q%5B%5D=test&search_field=all_fields&utf8=%E2%9C%93&view=gallery

Image Thumbnails 1-8 in grid is showing the following failed instance:
image-alt: Ensures <img> elements have alternate text or a role of none or presentation

Example for image 1
Path:
.row:nth-child(1) > .masonry.document.col-xs-6:nth-child(1) > .thumbnail > a > img

Snippet:
`<img src="/downloads/7s75dc44n?file=thumbnail">`

### Suggested Fix

Fix ONE of the following:

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by azle** [comment]
Getting the following error for validation:

CODE:
<img width="1029" alt="Screen Shot 2021-12-06 at 3 15 52 PM" src="https://user-images.githubusercontent.com/8353860/144937950-37c2cfd5-08d5-43d2-82e1-03eb2b354b59.png">

ERROR:
<img width="344" alt="Screen Shot 2021-12-06 at 3 16 28 PM" src="https://user-images.githubusercontent.com/8353860/144938011-b3ebbc9c-5d4c-4da2-9bb6-1ef9cea5ffaf.png">
